packages/@aws-cdk/core/test/test.cfn-parse.ts

```diff
@@ -0,0 +1,56 @@
+import { Test } from 'nodeunit';
+import { FromCloudFormation } from '../lib/cfn-parse';
+
+export = {
+  'FromCloudFormation class': {
```

Comment:
unit tests need to cover case when a valid creation policy is returned and assert that the values are as expected.

same for update policy.

Reply:
All of those are covered in the tests for `cloudformation-include`, so I don't see a reason to duplicate them here.